sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -193,7 +193,7 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
                 val actual = unindentAndTrim(
                   hiveResultString(df.queryExecution.executedPlan).mkString("\n"))
                 val expected = unindentAndTrim(output)
-                assert(actual.sorted === expected.sorted)
+                assert(actual === expected)
```

Comment:
Is there any other place like this?

Reply:
Yea, I think you can just `git grep -r "assert" . | grep "\.sorted"` and see if they are all necessary, e.g., remove them and run the tests, see if they pass. Looks not too many:

```bash
$ git grep -r "assert" . | grep "\.sorted" | wc -l
      74
```

Reply:
Even sorting is not necessary in other places, I don't think it makes sense to change those places here in the PR which is just a follow up https://github.com/apache/spark/pull/26813#discussion_r369743505

How about to open new JIRA for reviewing other places?

Reply:
Ya. What I meant was the other places in the original AQE PR.

Reply:
I see, let me check.

Reply:
This one sorts a text too https://github.com/apache/spark/blob/883ae331c36aa2279b163b87571997c8ffebfb6c/sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala#L422-L423

Reply:
Let's fix it too. I thought it's a list of string that for the result rows...